### DISCUSSION TITLE
why vitepress for docs? why not nextra?

### DISCUSSION THREAD

**Discussion by comerc** [ente-io/ente#990] [Enhancements]
https://github.com/shuding/nextra

**Comment by mnvr** [comment]
I'm not sure how to answer this :) Let me flip the question - why nextra for docs? why not vitepress?

While we're happy with Next, if anything, we're considering switching to Vite + React Router for our actual web app too. Next has its good parts, but is perhaps too big of a framework, and since we don't have any use case for RSCs currently, is perhaps overkill for us. 

For the docs specifically, so far vitepress has worked great. Of course, nextra may be even better, I shouldn't comment without trying things out, but there isn't any complain we have with vitepress so far for our use case.
